tests-hal-configuration/src/test/java/org/jboss/hal/testsuite/test/configuration/SubsystemsTest.java

```diff
@@ -75,6 +76,7 @@ public static void beforeClass() throws Exception {
     @Before
     public void setUp() throws Exception {
         browser.navigate().refresh();
+        Library.letsSleep(2000);
```

Comment:
It was added because in my point view the execution of code is probably fast after some tests. From this the page sometimes doesn't load and page stay blank. When you can think of some better solution just let me know.

Reply:
Thanks for the explanation and adding the comment. Did you try either removing `browser.navigate().refresh();` or adding conditional waiting - e.g. `waitModel().until().element(By.id(Ids.ROOT_CONTAINER)).is().present();`?

Reply:
I tried go step by step with test not running on virtual screen. I find that without `browser.navigate().refresh()` the should be fine. Only needed refresh of page is after `filter` test. So sleep method was removed.